### PR TITLE
Add Kotlin Script

### PR DESCRIPTION
**Pull request by d1snin**
## Adding a language Kotlin Script

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme

### CODE REVIEW

**Review by MrBrain295 — APPROVED** [review]
Looks good!

**Comment by Richienb** [comment]
What's the difference between Kotlin DSL and "regular" Kotlin?

**Comment by d1snin** [comment]
> What's the difference between Kotlin DSL and "regular" Kotlin?

In fact, this is a little different and different from the pull request code, just then I read somewhere that the build file `build.gradle` can also be in the `.gradle.kts` format, and it clearly stated about the use of a certain Kotlin DSL in this file, of course, I only later realized what it was, you can search for information about it and find more than I will say. By the way this should be renamed to something like Kotlin Script by my stupidity ¯\_(ツ)_/¯

**Comment by d1snin** [comment]
merge when

**Review by Richienb — CHANGES_REQUESTED** [review]
The language count should be updated

**Comment by d1snin** [comment]
done

File: readme.md [hunk 1]
```diff
@@ -458,6 +458,7 @@ Make sure to see [contributing.md](/contributing.md) for instructions on contrib
 * [Kitten](k/Kitten.ktn)
 * [Koberi C](k/Koberi%20C.koberice)
 * [Kotlin](k/Kotlin.kt)
+* [Kotlin Script](k/KotlinDSL.kts)
```
Comment (Richienb) [inline readme.md:461]: File name is not consistent with the actual file

**Review by Richienb — CHANGES_REQUESTED** [review]

**Review by Richienb — CHANGES_REQUESTED** [review]
The file name and language name in the readme need to agree on whether there should be a space between `Kotlin` and `Script`.

**Comment by d1snin** [comment]
thats literally horrible. but okay np
if you want

**Review by Richienb — APPROVED** [review]

**Comment by smokeytube** [comment]
@everyone 
image

**Pull request merged**